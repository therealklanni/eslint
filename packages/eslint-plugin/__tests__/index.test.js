'use strict'

const eslintPlugin = require('../lib')

describe('@therealklanni/eslint-plugin', () => {
  it('provides eslint configs', () => {
    expect(eslintPlugin.configs).toMatchSnapshot()
  })
})
