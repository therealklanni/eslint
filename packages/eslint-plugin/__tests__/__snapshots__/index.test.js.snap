// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@therealklanni/eslint-plugin provides eslint configs 1`] = `
Object {
  "base": Object {
    "extends": Array [
      "./configs/base",
    ],
  },
  "cli": Object {
    "extends": Array [
      "../configs/cli",
    ],
  },
  "jest": Object {
    "extends": Array [
      "../configs/jest",
    ],
  },
  "node": Object {
    "extends": Array [
      "../configs/node",
    ],
  },
  "prettier": Object {
    "extends": Array [
      "../configs/prettier",
    ],
  },
  "react": Object {
    "extends": Array [
      "../configs/react",
    ],
  },
  "typescript": Object {
    "extends": Array [
      "../configs/typescript",
    ],
  },
}
`;
